pkg/subctl/cmd/gather/logs.go

```diff
@@ -120,3 +124,39 @@ func findPods(clientSet kubernetes.Interface, byLabelSelector string) (*corev1.P
 
 	return pods, nil
 }
+
+func getPreviousLogs(pod *corev1.Pod, podLogOptions *corev1.PodLogOptions, info Info) error {
```

Comment:
Similar to above, this doesn't retrieve/return something. Also this mutates the `podLogOptions` param so pass by value.

```suggestion
func outputPreviousPodLog(pod *corev1.Pod, podLogOptions corev1.PodLogOptions, info Info) error {
```